ui/src/app/applications/components/application-status-panel/revision-metadata-panel.tsx

```diff
@@ -4,9 +4,54 @@ import {Timestamp} from '../../../shared/components/timestamp';
 import {services} from '../../../shared/services';
 
 export const RevisionMetadataPanel = (props: {appName: string; appNamespace: string; type: string; revision: string; versionId: number}) => {
-    if (props.type === 'helm') {
+    if (props.type === 'helm' || props.type === 'oci') {
         return <React.Fragment />;
     }
+    if (props.type === 'oci') {
+        return (
+            <DataLoader load={() => services.applications.ociMetadata(props.appName, props.appNamespace, props.revision, 0, props.versionId)} errorRenderer={() => <div />}>
+                {m => (
+                    <Tooltip
+                        popperOptions={{
+                            modifiers: {
+                                preventOverflow: {
+                                    enabled: false
+                                },
+                                hide: {
+                                    enabled: false
+                                },
+                                flip: {
+                                    enabled: false
+                                }
+                            }
+                        }}
+                        content={
+                            <span>
+                            {m.authors && <React.Fragment>Authored by {m.authors}</React.Fragment>}
```

Comment:
For these lint issues, try running `make lint-ui` or from the ui folder, run `yarn lint --fix` to auto fix the formatting issues.